specification/webhooks.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: default
+title: Providing Webhooks
+parent: The OpenAPI Specification Explained
+nav_order: 9
+---
+
+# Providing Webhooks
+
+Webhooks are a style of communication common in the API economy. Their purpose is to provide bi-directional communications from the API provider to the API consumer to support out-of-band events that do not fit into the pattern of REST APIs or Callbacks. Webhooks fulfill similar purposes to Callbacks in that they support transmitting information about asynchronous or long-running communications. The example of payments APIs discussed in the Callbacks topic also provides an example of the real-world implementations of Webhooks. Many payments API providers implement webhooks to communicate the result of payment operations. API consumers can implement a webhook at a URL of their choosing to receive out-of-band updates on a previously submitted, long-running request. This avoids implementing polling patterns, which many API providers find onerous to support.
```

Comment:
> Their purpose is to provide bi-directional communications 
this sentense may be missleadding, as  webhook is not bi directionnal ,

and it is not clear the differences between webhook and callback 

my own definition of webhook vs callback is given a server S , and a Client C1
given an action done by C1,  callback is a called from S to C1  ,

vs webhook supports more a brodcast pattern , meaning that given 2 clients C1, C2 an action done by C2 can trigger a webhook that C1 will receive . meaning the receiver of the call is not a reaction toward a previous call 
it could even be a webhook due to date . this is why it requires a registration mecanism , like a clock for instance

callback does not need it , url can even be provided at run time